### PR TITLE
Update MANUAL, add UCS

### PR DESCRIPTION
**Pull request by george0st**
Extend compaction list, add UCS

### CODE REVIEW

File: manual/MANUAL.adoc [hunk 1]
```diff
@@ -252,6 +252,18 @@ a|
 a|
 `{'class':'TimeWindowCompactionStrategy', 'compaction_window_size':'1', 'compaction_window_unit':'DAYS'}`
 
+|ucs
+a|
+`{'class':'UnifiedCompactionStrategy'}`
+
+|ucs,L4
+a|
+`{'class':'UnifiedCompactionStrategy', 'XXX':'L4'}`
```
Comment (rustyrazorblade) [inline manual/MANUAL.adoc:261]: This should be 

```
`{'class':'UnifiedCompactionStrategy', 'scaling_parameters':'L4'}`
```

File: manual/MANUAL.adoc [hunk 2]
```diff
@@ -252,6 +252,18 @@ a|
...
+
+|ucs,L4,L10
+a|
+`{'class':'UnifiedCompactionStrategy', 'XXX':'L4', 'YYY':'L10'}`
```
Comment (rustyrazorblade) [inline manual/MANUAL.adoc:265]: This should be 

```
`{'class':'UnifiedCompactionStrategy', 'scaling_parameters':'L4,L10'}`
```

**Review by rustyrazorblade — CHANGES_REQUESTED** [review]
Thanks for the manual update.  Please see my comments.

**Comment by rustyrazorblade** [comment]
Thanks for the PR!

**Comment by jrwest** [comment]
@george0st we are working to donate `easy-cass-stress` to the Apache foundation. Would you be willing to sign an [ICLA](https://apache.org/licenses/icla.pdf) and donate your contributions?

**Pull request merged**